style.css

```
@@ -0,0 +1,79 @@
+h3 {
+    margin-left: 15px;
+}
+.toDoList,
+.finishedList{
+    flex-grow: 1;
+    margin-left: 15px;
+    margin-right: 15px;
+    flex: 1;
```

`flex: 1;` 은 `flex-grow: 1; flex-shrink: 1; flex-basis: 0%;`과 똑같기 때문에 없어도 6행은 없어도 될 것 같아요!
```suggestion
    margin-left: 15px;
    margin-right: 15px;
    flex: 1;
```